docs/userguide/dependency_management.rst

```diff
@@ -341,4 +341,25 @@ In some cases, you might need to specify the minimum required python version.
 This is handled with the ``python_requires`` keyword supplied to ``setup.cfg``
 or ``setup.py``.
 
-Example WIP
```

Comment:
this went in as WIP here https://github.com/pypa/setuptools/commit/fdf51a0bee8de4cf0c10ccd2f94541242580fa0c#diff-d8e87bebd32b21394e170d3e1c0b80f4ab7760f535056951676be88b2d9994d9R49

I'm not sure if you had an example in progress you wanted here instead, @alvyjudy